source/module_base/opt_CG.h

```diff
@@ -49,27 +50,27 @@ class Opt_CG
         int &ifPD // if postive definit
     );
 
-    double get_residual() {return sqrt(this->gg);};
-    int get_iter() {return this->iter;}
+    double get_residual() {return sqrt(this->gg_);};
+    int get_iter() {return this->iter_;}
 
     // void ZEROS(double *x, int n)
     // {
     //     for (int i = 0; i < n; ++i) x[i] =0;
     // }
 
 private:
-    double dV = 1.;
-    int nx = 0; // length of the sulotion array x
-    int iter = 0; // number of iteration
-    double gg = 1000; // gradient dot gradient
-    double beta = 0.; // d = -g + beta * d
-    double eta = 0.01; // a constand used in HZ form
-    double *pdirect_old; // direction of last step
-    double *pgradient_old; // gradient, for meth=0, gradient is minus residual r.
+    double dV_ = 1.;
+    int nx_ = 0; // length of the sulotion array x
```

Comment:
Please format these files in module_base with .clang-format

Reply:
It's done.